shared/templates/audit_rules_syscall_events/bash.template

```diff
@@ -0,0 +1,19 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
+
+# Include source function library.
+. /usr/share/scap-security-guide/remediation_functions
```

Comment:
We are no longer use this method for shared functions.

Please use [Jinja macros](https://complianceascode.readthedocs.io/en/latest/jinja_macros/bash.html).